backtrace-library/src/main/java/backtraceio/library/common/DeviceAttributesHelper.java

```diff
@@ -187,20 +186,23 @@ private WifiStatus getWifiStatus() {
      * @return is power saving mode activated
      */
     // TODO: replace bool to enum
+    /*
     private boolean isPowerSavingMode() {
         if (Build.VERSION.SDK_INT < 21) {
             return false;
         }
         PowerManager powerManager = (PowerManager) this.context.getSystemService(Context
                 .POWER_SERVICE);
         return powerManager.isPowerSaveMode();
-    }
+    }*/
 
+    // Try removing this since it's not the correct way to do this
     /**
      * Get a battery level in float value (from 0.0 to 1.0) or -1 if error occurs
      *
      * @return battery level from 0.0 to 1.0 or -1 if error occurs
      */
+     /*
     private float getBatteryLevel() {
```

Comment:
If we trust the new method from `BatteryStateHelper` to do this now, we can just remove this, no need to comment it out. But only if we're okay with users no longer having this information by default and needing to explicitly enable `BatteryStateHelper` to get this information now.